server/contracts.go

```diff
@@ -0,0 +1,63 @@
+package server
+
+import (
+	"fmt"
+	"io/ioutil"
+	"regexp"
+
+	emulator "github.com/onflow/flow-emulator"
+	"github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk/templates"
+	"github.com/onflow/flow-nft/lib/go/contracts"
+)
+
+var ftAddress = flow.HexToAddress("ee82856bf20e2aa6")
```

Comment:
I would try not to hardcode this. Can you try using this instead:  https://github.com/onflow/flow-go/blob/3932592e17a289880b693e4e57b9044d09dc194a/fvm/bootstrap.go#L927

Reply:
Ah! Yes, good idea. Will change to be the fvm address